### ISSUE TITLE
TypeError: Cannot read property 'name' of null

### ISSUE THREAD

**Issue by shockey** [supnate/rekit-core#1]
## Context

Created a new Rekit app and was trying to clean out the boilerplate stuff.

Ran the following commands (gleaned from my `.zsh_history`:

```sh
: 1516848874:0;rekit rm component home/RedditList
: 1516848882:0;rekit rm component home/TestPage
: 1516848950:0;rekit rm action home/fetchRedditReactjsList
: 1516848969:0;rekit rm action home/counterPlusOne
: 1516848974:0;rekit rm action home/counterMinusOne
: 1516848993:0;rekit rm action home/resetCounter
: 1516849642:0;rekit rm action home/counterMinusOne
```

The last command caused a TypeError:

```sh
➜  frontend git:(master) ✗ rekit rm action home/counterMinusOne  
./frontend/node_modules/rekit-core/core/refactor/array.js:185
      const toRemove = _.find(node.init.elements, ele => ele.name === identifierName);
                                                             ^

TypeError: Cannot read property 'name' of null
    at VariableDeclarator._.find.ele (./frontend/node_modules/rekit-core/core/refactor/array.js:185:62)
    at baseFindIndex (./frontend/node_modules/lodash/lodash.js:823:11)
    at findIndex (./frontend/node_modules/lodash/lodash.js:7275:14)
    at Function.find (./frontend/node_modules/lodash/lodash.js:5098:21)
    at VariableDeclarator (./frontend/node_modules/rekit-core/core/refactor/array.js:185:26)
    at NodePath._call (./frontend/node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (./frontend/node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (./frontend/node_modules/babel-traverse/lib/path/context.js:105:12)
    at TraversalContext.visitQueue (./frontend/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitMultiple (./frontend/node_modules/babel-traverse/lib/context.js:103:17)
```

### Versions

```sh
➜  frontend git:(master) ✗ node -v                             
v8.9.1
➜  frontend git:(master) ✗ npm ls | grep rekit
├─┬ rekit-core@2.2.0
├─┬ rekit-portal@2.1.3
```

**Comment by supnate** [comment]
Hi @shockey , thanks for reporting this. To create a clean app you can use:
```
rekit create app --clean
```

I will look into the issue later. BTW, I suggest using Rekit Portal to managing elements.
And, `rekit-core` has been moved to `rekit` which is a monorepo. You could post issues there instead.

**Comment by supnate** [comment]
Fixed by https://github.com/supnate/rekit/commit/3069c4640ddcb610957e7836494640d6a1cfcac2

And published rekit-core@2.2.1 to include the fix.
Creating a new app should have no problem now with cli.

Thanks again for reporting, it's really a critical issue.

**Comment by shockey** [comment]
Thanks for the quick fix @supnate! I didn't know about the `--clean` flag, that was helpful.

I didn't notice the monorepo, my apologies! I'll open future issues against `supnate/rekit` 👍
